File: demos/java/src/miscellaneous/Rectangle.java
package miscellaneous;

public class Rectangle {
    private int borderWidth;
    private Point topLeft;
    private Point bottomRight;
    private Color backgroundColor;
    private Color borderColor;
    private String text;

    public Rectangle(int borderWidth, Point topLeft, Point bottomRight, Color backgroundColor, Color borderColor,
            String text) {
        this.borderWidth = borderWidth;
        this.topLeft = topLeft;
        this.bottomRight = bottomRight;
        this.backgroundColor = backgroundColor;
        this.borderColor = borderColor;
        this.text = text;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public Color getBorderColor() {
        return borderColor;
    }

    public void setBorderColor(Color borderColor) {
        this.borderColor = borderColor;
    }

    public Color getBackgroundColor() {
        return backgroundColor;
    }

    public void setBackgroundColor(Color backgroundColor) {
        this.backgroundColor = backgroundColor;
    }

    public int getBorderWidth() {
        return borderWidth;
    }

    public Point getBottomRight() {
        return bottomRight;
    }

    public void setBottomRight(Point bottomRight) {
        this.bottomRight = bottomRight;
    }

    public Point getTopLeft() {
        return topLeft;
    }

    public void setTopLeft(Point topLeft) {
        this.topLeft = topLeft;
    }

    public void setBorderWidth(int borderWidth) {
        this.borderWidth = borderWidth;
    }

    @Override
    public String toString() {
        return text;
    }
}

File: demos/csharp/src/miscellaneous/Rectangle.cs
namespace ExampleCsharpProject
{
    class Rectangle
    {
        public int? borderWidth;
        public Point? topLeft;
        public Point? bottomRight;
        public Color? backgroundColor;
        public Color? borderColor;
        public string text;

        public Rectangle(int _borderWidth, Point _topLeft, Point _bottomRight, Color _backgroundColor, Color _borderColor, string _text)
         => (borderWidth, topLeft, bottomRight, backgroundColor, borderColor, text) = (_borderWidth, _topLeft, _bottomRight, _backgroundColor, _borderColor, _text);

        public override string ToString() => text;
    }
}